package-lock.json

```diff
@@ -1,6484 +1,7 @@
 {
   "name": "@autorest/csharp",
-  "lockfileVersion": 2,
```

Comment:
This looks wrong. Are you using lockfileVersion: 1?

Reply:
I am not sure - some dpg changes updates the version of autorest.testserver, and I run `npm install`, this comes out automatically